InfiniteLoopLMSSQL/src/com/st/lmssql/models/Author.java

```diff
@@ -0,0 +1,34 @@
+package com.st.lmssql.models;
+
+public class Author {
+	private int authorId;
+	private String authorName;
+	
+	public Author() {}
+	
+	public Author(int authorId, String authorName) {
+		this.authorId = authorId;
+		this.authorName = authorName;
+	}
+
+	public int getAuthorId() {
+		return authorId;
+	}
+
+	public void setAuthorId(int authorId) {
+		this.authorId = authorId;
+	}
+
+	public String getAuthorName() {
+		return authorName;
+	}
+
+	public void setAuthorName(String authorName) {
+		this.authorName = authorName;
+	}
+
+	@Override
+	public String toString() {

```

Comment:
You may want to override the equals and hashCode method. One reason you would want to do this is when you are comparing objects, like in your tests.